yaml/csr.yaml.m4

```diff
@@ -3096,6 +3101,7 @@ ifelse(eval(CLIC != 0), 1, [[[
   address: 0x305
   privilege_mode: M
   rv32:
+ifelse(eval(VERIF_HEADER != 0), 1, [[[
```

Comment:
Can't this ifdef be prevented by using another legalize function on bits 31-6 combined?

Reply:
The idea is that the verification headers may benefit from having the base bitfield split in two, while the sw headers are likely better off with one base field.